static/app/components/menuListItem.tsx

```diff
@@ -235,6 +235,7 @@ export const InnerWrap = styled('div', {
 const ContentWrap = styled('div')<{isFocused: boolean; showDivider: boolean}>`
   position: relative;
   width: 100%;
+  min-width: 0;
```

Comment:
`ContentWrap` is a flex item. By default, flex items have `min-width: auto`, which generally evaluate to the inner content width. Explicitly setting `min-width: 0` means that `ContentWrap` can now shrink to occupy the remaining space in its container, and no more.